ide/languages.toml/nbproject/project.properties

```diff
@@ -0,0 +1,19 @@
+is.eager=true
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
value above license

Reply:
Yeah, that's bad. I need to patch the UI action that put it there one day.